Darwin/StitchCore/StitchCoreTests/Core/StitchAppClientIntegrationAuthTests.swift

```diff
@@ -1,4 +1,9 @@
 // swiftlint:disable function_body_length
+// swiftlint:disable type_body_length
+// swiftlint:disable empty_parentheses_with_trailing_closure
+// swiftlint:disable cyclomatic_complexity
+// swiftlint:disable line_length
+
```

Comment:
We should not be disabling this for the whole file. These should be disabled where necessary. Also `empty_parentheses_with_trailing_closure` should not be disabled.

Reply:
I think it's fine to do file wide disables in test code IMO, but I agree that rule should not be disabled